source/openapi-admin-v3.yaml

```diff
@@ -4387,6 +4387,137 @@ paths:
         "204":
           description: No Content
 
+  "/groups/{groupId}/apps/{appId}/edge":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+
+    get:
+      tags:
+        - edge-server
+      operationId: adminGetAllEdgeServerInstances
+      summary: List Edge Servers
+      description: |-
+        List all [Edge Server](https://www.mongodb.com/docs/atlas/app-services/edge-server/) 
+        instances for the given App.
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  edge_servers:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/EdgeServerInstance"
+
+    post:
+      tags:
+        - edge-server
+      operationId: adminCreateEdgeServerInstance
+      summary: Create an Edge Server instance
+      description: Create a new Edge Server instance.
+      requestBody:
+        required: true
+        content:
+            application/json:
+              schema:
+                type: object
+                required: ["name"]
+                properties:
+                  name:
+                    type: string
+                    example: Edge-Store-42
+                    description: A human-readable name for the Edge Server instance.
+                    required: true
+      responses:
+        "201":
```

Comment:
I know we're crunched for time on this, but in general, should we be doc'ing the errors too?